tests/mocks/fake_virtual_accounts.json

```diff
@@ -0,0 +1,21 @@
+{
+  "amount_paid": 0,
+  "created_at": 1506717714,
+  "customer_id": null,
+  "description": null,
+  "entity": "virtual_account",
+  "id": "fake_virtual_accounts_id",
+  "name": "Super Potato",
+  "notes": [],
+  "receivers":
+  [
+    {
+      "account_number": "RZRPAY00A7VRC4HQM",
+      "entity": "bank_account",
+      "id": "ba_8jHPWHXbSc9xc7",
+      "ifsc": "KKBK0000958",
+      "name": "Super Potato"
+    }
+  ],
+  "status": "closed"
```

Comment:
Order of params seems incorrect.

Reply:
Also, I think it would be better if we tried to use the same examples across libraries.